docs/application/vstools/Tizen/import.md

```diff
@@ -0,0 +1,25 @@
+# Importing Tizen Studio Projects Into Visual Studio
+
+The following sections explain how to use Visual Studio Extension for Tizen to import your projects created in Tizen Studio to Visual Studio.
+
+> [!NOTE]
+> Ensure that the Tizen Studio project was exported to CLI before importing to Visual Studio
+
+### Add Tizen Native project
```

Comment:
I think this should be a second heading, since there's no second heading before it....
```suggestion
## Add Tizen Native project
```

Reply:
Thanks @safir-srbd , adding changes to new patch

Reply:
@alka-sethi You seemed to have removed this part from the md file. Is there any specific reason for that?

Reply:
Hi @safir-srbd , The lines were unnecessary, hence removed them

Reply:
@alka-sethi Thanks for letting me know!